newIDE/app/src/SceneEditor/index.js

```diff
@@ -1309,6 +1310,18 @@ export default class SceneEditor extends React.Component<Props, State> {
               }
             }}
             hotReloadPreviewButtonProps={this.props.hotReloadPreviewButtonProps}
+            onComputeAllVariableNames={() =>
+              this.state.variablesEditedObject
```

Comment:
This will not work I think.
`variablesEditedObject` is null here. Because you're editing an instance variables (`this.state.variablesEditedInstance` is not null) but not an object (so `this.state.variablesEditedObject` is null).

If you want the object of an instance, you have to find it, like done in "onEditObjectVariables" just above:

Get the object name associated to the instance: `const associatedObjectName = variablesEditedInstance.getObjectName()`
Use it to get the object: 
`getObjectByName(project, layout, associatedObjectName)`
Finally, if object is not null, you can feed it to `findAllObjectVariables` :)

This is because we don't keep "links" between instances and objects. The only link is the object name, so we have to find to which object it refers to (helped by `getObjectByName` - you can see btw in the codebase and see it's used in LayerRenderer - i.e: in the scene renderer :) Because when we iterate on instance, we must get on the first time the associated object to know which renderer we must create).